ncm-ipmi/src/test/resources/ipmi.pan

```diff
@@ -2,7 +2,9 @@ object template ipmi;
 
 prefix "/software/components/ipmi";
```

Comment:
bind the schema here as well, to test the schema

Reply:
good call

Reply:
Done.